block/src/Pos/Block/Verification.hs

```diff
@@ -0,0 +1,158 @@
+{-# LANGUAGE TypeFamilies  #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- This module could be in core, but 'verifySscPayload' is kind of
```

Comment:
Stale comment? Is this module to do with SSC?

Reply:
The comment is highly relevant.